receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -37,6 +37,8 @@ import (
 
 const (
 	targetMetricName = "target_info"
+	traceIDKey       = "trace_id"
+	spanIDKey        = "span_id"
```

Comment:
Are these keys coming from some sort of specification? If yes, please document from where.

Reply:
Done, added in Readme and we are acknowledging `trace_id or traceid` from labels and setting it to otlp traceid